dbt/adapters/databricks/python_models/run_tracking.py

```diff
@@ -8,20 +8,20 @@
 
 
 class PythonRunTracker(object):
-    _run_ids: Set[str] = set()
+    _run_ids: Set[int] = set()
```

Comment:
I'm not seeing how this is used, but is it intended to move to int?

Reply:
this is because with migrating to the workspace client, we get a real type, as opposed to strings.